packages/connectkit/src/components/ConnectKit.tsx

```diff
@@ -21,6 +21,8 @@ import { ThemeProvider } from 'styled-components';
 import { useThemeFont } from '../hooks/useGoogleFont';
 import { useAccount, useNetwork } from 'wagmi';
 import { SIWEContext } from './../siwe';
+import { getGlobalChains } from '../defaultClient';
```

Comment:
unnecessary import?

Reply:
Seems like its used here: 
```
initialChainId: getGlobalChains()[0]?.id,
```

Reply:
NVM just pulled the latest changes. Will remove